sentry-android-replay/src/main/java/io/sentry/android/replay/viewhierarchy/ViewHierarchyNode.kt

```diff
@@ -47,11 +47,11 @@ data class ViewHierarchyNode(
             return actualPosition.intersects(screen.left, screen.top, screen.right, screen.bottom)
         }
 
-        fun adjustAlpha(color: Int): Int {
+        fun Int.toOpaque(): Int {
             val alpha = 255
-            val red = Color.red(color)
-            val green = Color.green(color)
-            val blue = Color.blue(color)
+            val red = Color.red(this)
+            val green = Color.green(this)
+            val blue = Color.blue(this)
             return Color.argb(alpha, red, green, blue)
```

Comment:
Future improvement in case you need low level optimizations: do some bit wise composition, like [it's done here](https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/core/java/com/android/internal/graphics/ColorUtils.java;l=372-379;drc=dde365777a3364f5ec32975fcd5af7f0fd681c80).